rhdevdocs-2073 Update the getting started experience for ODC (#24170)
openshift/openshift-docs: web_console/odc-about-developer-perspective.adoc

toc::[]

The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective.
[NOTE]
====

> The default web console perspective is shown depending on the user's role. (If the user is recognised as a developer, a Developer perspective will be selected by default.)

Reviewer: user's role>the role of the user
Remove the parenthesis for the second sentence.
a Developer perspective will be selected by default.> the **Developer** perspective is displayed by default.

Author: ```suggestion
The default web console perspective is shown depending on the role of the user. If the user is recognised as a developer, the *Developer* perspective is displayed by default.
```

> The default web console perspective is shown depending on the role of the user. If the user is recognised as a developer, the *Developer* perspective is displayed by default.

Reviewer 2: 1. The first sentence could be interpreted to mean that the default web console is not shown if the user has a certain role. To be more explicit, you could say something like the following instead:

"The default web console perspective that is shown depends on the role of the user."

2. For the second sentence, I think it would be useful to be specific about the cluster role that is required for the developer perspective to be displayed by default. You could also swap the sentence elements around in order to remove the need for a comma. For example, you could say something like the following. Please replace `<role>`with whichever role causes the UI to default to the Developer perspective:

"The developer perspective is displayed by default if the user has the `<role>` role."

Author: 1. it is a weird nomenclature that the docs follow. "perspective"
2. we have decided to not elab on the roles here.

thanks for the review, and wdyt about this? vvvv

```suggestion
The default web console perspective that is shown depends on the role of the user. The *Developer* perspective is displayed by default if the user is recognised as a developer.
```

Reviewer 2: @pmacko1 that suggestion looks good to me.
